### PR TITLE
[RFR] Fix FilterForm typo

### LINKED ISSUE #633: maddingBottom  -- a typo?
https://github.com/marmelab/admin-on-rest/blob/6bdb56dd6b41035f53b875ee10b43da49e0dc55c/src/mui/list/FilterForm.js#L15

### PR DESCRIPTION
**Pull request by djhi**
Supercedes #635 
Fixes #633

### CODE REVIEW

File: src/mui/list/FilterForm.js [hunk 1]
```diff
@@ -25,11 +25,11 @@ export class FilterForm extends Component {
             .filter(filterElement =>
                 filterElement.props.alwaysOn ||
                 displayedFilters[filterElement.props.source] ||
-                typeof initialValues[filterElement.props.source] !== 'undefined'
+                typeof initialValues[filterElement.props.source] !== 'undefined',
```
Comment (zyhou) [inline src/mui/list/FilterForm.js]: Sure about `'` ?

Comment (djhi) [inline src/mui/list/FilterForm.js]: Yup, it has been inserted by eslint fix

File: src/mui/list/FilterForm.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ export class FilterForm extends Component {
                                 record={emptyRecord}
                             />
                         </div>
-                    </div>
+                    </div>,
```
Comment (ThieryMichel) [inline src/mui/list/FilterForm.js]: Are you sure it won't cause syntax error ?

Comment (ThieryMichel) [inline src/mui/list/FilterForm.js]: checked it, it is a syntax error.

Comment (ThieryMichel) [inline src/mui/list/FilterForm.js]: nvm, but it is perturbing

Comment (djhi) [inline src/mui/list/FilterForm.js]: Well, that's why I do recommend we move to prettier => no more questions !

**Pull request merged**